### DISCUSSION TITLE
Should we/when should we exclude chrM/X/Y in our analysis?

### DISCUSSION THREAD

**Discussion by hsun3163** [StatFunGen/xqtl-protocol#165] [Default analysis setting]
In lots of times the analysis only take autosomal chromosome e.g. GRM module. But also in a lot of time the imput data may contains sex chromosome as well. So in our pipeline, when is a good time to exclude the sex chromosome?

A safe time to exclude them I think is when we partitioned the whole genome plink into plink per chrom.

p.s. As GRM for  chrM/X/Y , APEX association scan can not be done on these chromosome.

**Comment by gaow** [comment]
No we only consider chroms 1-22.

**Reply by gaow** [reply]
you might want to add an interface somewhere to remove other chroms... ?
